src/spam-protection/spam-protection-action-creator.ts

```diff
@@ -48,17 +44,37 @@ export default class SpamProtectionActionCreator {
         );
     }
 
-    execute(): ThunkAction<SpamProtectionAction, InternalCheckoutSelectors> {
+    executeCheckoutSpamProtection(): ThunkAction<SpamProtectionAction> {
```

Comment:
`verifyCheckoutSpamProtection` so that it mirrors the name of the action you're emitting?